src/Analyzer.py

```diff
@@ -105,6 +105,14 @@ def __init__(self, Data_to_analyze, AcceptableGeneralMismatch, FIDsNotToBeAnalyz
             else:
                 continue
 
+    def columnAutoWidth(self, wbSheet, *val):
```

Comment:
Func can be refactored to have its call after filling the whole column with data. Save each cell payload length in separate var and then with respect to the longest word - set the length. Then we will have the function call once for the whole sheets and not for the each string of payload.